sdk/textanalytics/azure-ai-textanalytics/pom.xml

```diff
@@ -67,7 +67,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.8.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core-test;dependency} -->
+      <version>1.7.3</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
```

Comment:
do you need to change it in the version_client file too?

Reply:
We can leave unrelease_com in the version_client